runners/pom.xml

```diff
@@ -39,6 +39,7 @@
     <module>direct-java</module>
     <module>flink</module>
     <module>spark</module>
+    <module>gearpump</module>
```

Comment:
Since gearpump requires Java 8, you can use maven profiles to only include this module when the JDK is 8 or higher. Or perhaps it is just certain tests that require it, so you could limit the effect of the profile to exactly what needs to be disabled. We do this with the `sdks/java/java8tests` module, if you want an example.